app/routes.py

```diff
@@ -63,3 +64,37 @@ def post_kml():
         ),
         201
     )
+
+
+@app.route('/kml/<kml_admin_id>', methods=['GET'])
+def get_id(kml_admin_id):
+    logger.debug("entering route /kml/%s", kml_admin_id)
```

Comment:
This logging could be done in a more generic way, see https://github.com/geoadmin/template-service-flask/blob/7ac89b5cadd9ca0f5a00f2ff4be79bb3630b4f6b/app/__init__.py#L27 and https://github.com/geoadmin/template-service-flask/blob/7ac89b5cadd9ca0f5a00f2ff4be79bb3630b4f6b/app/__init__.py#L58

Reply:
I'm adding both the pre-request log and the post-request log